### DISCUSSION TITLE
volar 配合 vue-vben-admin 使用时，CPU占用非常高，很卡，代码智能提醒要几秒才能显示

### DISCUSSION THREAD

**Discussion by KiteWorld** [vbenjs/vue-vben-admin#1578] [Q&A]
volar 配合 vue-vben-admin 使用时，CPU占用非常高，很卡，代码智能提醒要几秒才能显示，有遇到同样问题的朋友吗？求助是否感谢

**Comment by i7eo** [comment]
有遇到，这个问题不是vben的问题，是一系列插件的问题（vite、volar、vscode插件）。
1. 要么换电脑
2. 要么等更新

我选择换了电脑😼
